### PR TITLE
Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-ansible-role-with-test-user.git`](https://github.com/cisagov/skeleton-ansible-role-with-test-user.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +75,13 @@ Additionally, the following variables may be useful to override:
 - `geoip_local_path` - The directory to extract the database into.
 - `geoip_local_file` - The filename used to store the downloaded database.
 
+<!--
+| Variable | Description | Default | Required |
+|----------|-------------|---------|----------|
+| optional_variable | Describe its purpose. | `default_value` | No |
+| required_variable | Describe its purpose. | n/a | Yes |
+-->
+
```
Comment (jsf9k) [inline README.md]: The role variable descriptions should be reworked into tabular format.

**Review by jsf9k — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dav3r) [reply README.md]: Please enjoy aa8a916 with these changes.

**Review by dav3r — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -59,18 +62,15 @@ None.
...
-- `geoip_local_file` - The filename used to store the downloaded database.
+| Variable | Description | Default | Required |
+|----------|-------------|---------|----------|
+| geoip_local_file | The filename used to store the downloaded database. | `GeoIP2-City.tar.gz` | No |
+| geoip_local_path | The directory to extract the database into. | `/usr/local/share/GeoIP/` | No |
+| maxmind_edition | The database edition to install. | `GeoIP2-City` | No |
+| maxmind_license_key | The MaxMind GeoIP2 license key to use when accessing the MaxMind servers. | n/a | Yes |
+| maxmind_suffix_checksum | The suffix of the database checksum file to be downloaded. | `tar.gz.md5` | No |
+| maxmind_suffix_file | The suffix of the database file to be downloaded. | `tar.gz` | No |
+| maxmind_url_format | The format of the MaxMind URL, where the first `%s` represents `maxmind_url_base`, the second `%s` represents `maxmind_edition`, the third `%s` represents `maxmind_suffix_file` or `maxmind_suffix_checksum`, and the fourth `%s` represents `maxmind_license_key`. | `%s?edition_id=%s&suffix=%s&license_key=%s` | No |
```
Comment (dav3r) [inline README.md:73]: @mcdonnnj - Please feel free to edit any of the descriptions here.  Some were already present and some I created just now.

**Review by jsf9k — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (mcdonnnj) [reply README.md:73]: I think these are ok as things stand. A number of these should be moved to a file in the `vars/` directory and pulled in with `ansible.builtin.include_vars` as they're generally not meant to be changed. I've created #10 to reflect this work.

**Review by mcdonnnj — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jsf9k) [reply README.md:73]: `defaults` not `vars`!  I also commented in #10.

Comment (jsf9k) [reply README.md:73]: Resolved in #10.  @mcdonnnj was right!

**Pull request merged**